### ISSUE TITLE
[Bug?] Invalid render when using empty lines

### ISSUE THREAD

**Issue by 0xF6** [jonathanvdc/Pixie#1]
With empty lines, line number is render not correct~

![image](https://user-images.githubusercontent.com/13326808/60473029-feb5a500-9c73-11e9-8635-1ea6fc25b729.png)

Source document:
```csharp
public static class Program
{

    public Program()
    { 
    
    }

}
```

Without use of empty lines, drawing is correct:

![image](https://user-images.githubusercontent.com/13326808/60473005-e6de2100-9c73-11e9-9aef-b9b22641c100.png)

Tested on:
`win10, .net core 2 and 3`

Gits: [peeew~](https://gist.github.com/0xF6/f4f75867ebf7af19953d82f9a8c7d3aa)

**Comment by jonathanvdc** [comment]
Hi @0xF6! Thanks for trying out Pixie. I'll try to get this bug fixed as soon as possible.

**Comment by jonathanvdc** [comment]
I pushed a fix for the bug you encountered and released a new version of the Pixie package (0.1.9). You should be able to download it once NuGet has indexed the package, which usually takes no more than ten minutes.

Thanks a bunch for your detailed bug report. It was super helpful.

I'll now close this issue, if that's alright with you.

**Comment by 0xF6** [comment]
thanks!
